try {
    throw new Error("hell")
}catch (e) {
    
}
